setup.py

```diff
@@ -74,6 +74,10 @@
                      'dltorch': [
                          'torch==1.0.1',
                      ],
+                     'dlonnx': [
+                         'mxnet',
+                         'onnx'
```

Comment:
Should we keep it like this? I am asking since ONNX is not supported by the latest version of MXNet